### DISCUSSION TITLE
Fox-Glove bridge for ros2 foxy

### DISCUSSION THREAD

**Discussion by arpitpara** [foxglove/community#1004] [General]
Hi
I am trying to build the repo from source for ros2 foxy.
It is having lots of errors.
is it possible to do so?

**Comment by defunctzombie** [comment]
It is unlikely to lead to success since various APIs we use in that codebase are likely changed because Foxy is so old. ROS2 Foxy is EOL Since June of 2023: https://docs.ros.org/en/foxy/Releases.html
